### PR TITLE
More clean ups

### PR DESCRIPTION
**Pull request by auvipy**

### CODE REVIEW

**Comment by auvipy** [comment]
@justquick could you review this pr and give me suggestions?

**Comment by pmourelle** [comment]
Hey @auvipy, there're some conflicts and travis failing? Could you take a look at those and solve them?
Thanks.

File: docs/source/installation.rst [hunk 1]
```diff
@@ -40,29 +40,14 @@ Add the activity urls to your urlconf
 
 .. code-block:: python
 
-    urlpatterns = patterns('',
+    urlpatterns = [
         ...
         ('^activity/', include('actstream.urls')),
```
Comment (pmourelle) [inline docs/source/installation.rst]: `url(...),`?

**Comment by pmourelle** [comment]
Nice clean up btw. 👍

**Comment by auvipy** [comment]
thanks for your time for review! i will fix em ASAP

**Comment by subho007** [comment]
any idea when this will be merged ? too many conflicting file, and I cannot test this in my system

**Comment by auvipy** [comment]
I will complete this 1 week later

**Comment by pmourelle** [comment]
@auvipy do you have merge permissions? (and eventually for deploy?). I've had trying to reach Justin since a while and no luck so far.

**Comment by auvipy** [comment]
yes i have merge permissions

**Comment by auvipy** [comment]
6.5 is on pypi

**Pull request merged**